lib/amqp_client.js

```diff
@@ -192,18 +192,22 @@ AMQPClient.prototype.createSender = function(address, options) {
     throw new Error('Must connect before creating links');
   }
 
-  address = address || this._defaultQueue;
+  address = u.parseLinkAddress(address || this._defaultQueue);
```

Comment:
They do - receivers for EH typically look like _ehname_/ConsumerGroups/$default/Partitions/1